sky/services/media/ios/media_player_impl.mm

```diff
@@ -58,6 +60,22 @@ - (BOOL)seekTo:(NSTimeInterval)interval {
   return [_player playAtTime:_player.deviceCurrentTime + interval];
 }
 
+- (void)setVolume:(double)volume {
+  if (volume > 1.0) {
```

Comment:
media.mojom is based on the Android interface https://github.com/flutter/engine/commit/04ff30d78b8e1613fccfb4e3c9502bc337ad10da#diff-9f13ad3e7c279cc26b97b442f2d9b81aR118 which takes scalars in the 0.0 to 1.0 range